uninstall.php

```diff
@@ -0,0 +1,173 @@
+<?php
+/**
+ * Distributor uninstall script.
+ *
+ * @package distributor
+ * @since   x.x.x
+ */
+
+// phpcs:disable
```

Comment:
```suggestion
```

Rather than disabling all styles, just disable them on a line by line basis with a comment explaining why. It will help prevent future devs (probably ourselves) from introducing errors.